<launch>
    <node name="camera_publisher" pkg="camera_publisher_pkg" type="camera_publisher.py" output="screen">
        <param name="camera_device" value="/dev/video0" />
        <param name="frame_rate" value="30" />
        <param name="resolution" value="640x480" />
        <param name="enable_display" value="true" />
        <param name="enable_save" value="true" />
        <env name="PYTHONUNBUFFERED" value="1" />
    </node>
</launch>